src/ex4/server/routes/api.js

```diff
@@ -1 +1,48 @@
-// Define your endpoints here (this is your "controller file")
\ No newline at end of file
+// Define your endpoints here (this is your "controller file")
+import { Router } from "express";
+import express from "express";
+import ItemManager from "../services/item_manager.js";
```

Comment:
creating a line of separation between the imports and the start of the logic (consts in this case) will make the code tidier